src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/ProgressBar.tsx

```diff
@@ -40,9 +41,14 @@ const checkEntities = (clinicalEntities: ProgressBarProps['clinicalEntities']) =
 type ProgressBarProps = {
 	clinicalEntities: ClinicalSubmission['clinicalEntities'] | undefined;
 	clinicalState: ClinicalSubmissionQuery['clinicalSubmissions']['state'] | undefined;
+	approvalBarWidth?: number;
 };
 
-const ProgressBar: FC<ProgressBarProps> = ({ clinicalEntities, clinicalState }) => {
+const ProgressBar: FC<ProgressBarProps> = ({
```

Comment:
Going to take a wager, and assume the use of FC here is so you can pass children?
I recall ages ago using the FC type was discouraged, because specifying the type explicitly leads to less errors.

you could use `PropsWithChildren<Props>` if children is all you need

Reply:
FC<ProgressBarProps> is not my doing. In platform-ui this is `ComponentType<ProgressBarProps>`, which works fine in RDPC as well.
`PropsWithChildren<ProgressBarProps>` does not work for me.

Reply:
you're right. once again I feel into the "it's green, new code" trap 😂

in either case, at risk of stating the obvious and just to verify a suspicion: the replacement for 
`const X: FC<Props> = props =>...` is `const X = (props: PropsWithChildren<Props>) =>...`, which in turn lets TS infer the return type (a much better way to use TS in React)

is that how you did it?

i.e. ' const X: PropsWithChildren<Props> =...` wouldn't work because X is meant to be a component, not just its props.

Reply:
OK I see. `const X = (props: PropsWithChildren<Props>) =>` does work.

Reply:
glad to hear! thanks for helping leave these systems better than we found them 🙏